[BUG] Cannot find name 'AnchorPositioningPolyfillOptions' in v0.3.0

**Describe the bug**
Running `npx -p typescript tsc index.mjs --declaration --allowJs --emitDeclarationOnly` end in the following error:

```bash
node_modules/@oddbird/css-anchor-positioning/dist/polyfill.d.ts(17,72): error TS2304: Cannot find name 'AnchorPositioningPolyfillOptions'.
tsc exited with code 2
```

Running the same command worked fine with v0.2.0

**Additional context**
The `tsconfig.json`

```json
{
	"compilerOptions": {
		"target": "ES2020",
		"experimentalDecorators": true,
		"useDefineForClassFields": false,
		"module": "ESNext",
		"lib": ["ES2020", "DOM", "DOM.Iterable"],
		"skipLibCheck": true,

		/* Bundler mode */
		"moduleResolution": "bundler",
		"allowImportingTsExtensions": true,
		"resolveJsonModule": true,
		"isolatedModules": true,
		"noEmit": true,

		/* Linting */
		"strict": true,
		"noUnusedLocals": true,
		"noUnusedParameters": true,
		"noFallthroughCasesInSwitch": true,

		/* Project */
		"declaration": true,
		"declarationMap": true,
		"sourceMap": true,
		"inlineSources": true,
		"rootDir": "./src",
		"outDir": "./dist/",

		/* Custom */
		"noImplicitReturns": true,
		"noImplicitAny": true,
		"noImplicitThis": true,
		"allowSyntheticDefaultImports": true,
		"forceConsistentCasingInFileNames": true,
		"noImplicitOverride": true,
		"plugins": [
			{
				"name": "ts-lit-plugin",
				"strict": true
			}
		]
	},
	"include": ["src"]
}
```

--- a/src/@types/global.d.ts
+++ b/src/@types/global.d.ts
@@ -1,23 +1,8 @@
+import { type AnchorPositioningPolyfillOptions } from '../polyfill.ts';
+
 export {};
 
 declare global {
-  interface AnchorPositioningPolyfillOptions {
-    // Whether to use `requestAnimationFrame()` when updating target elements’
-    // positions
-    useAnimationFrame?: boolean;
-
-    // An array of explicitly targeted elements to polyfill
-    elements?: HTMLElement[];
-
-    // Whether to exclude elements with eligible inline styles. When not defined
-    // or set to `false`, the polyfill will be applied to all elements that have
-    // eligible inline styles, regardless of whether the `elements` option is
-    // defined. When set to `true`, elements with eligible inline styles listed
-    // in the `elements` option will still be polyfilled, but no other elements
-    // in the document will be implicitly polyfilled.
-    excludeInlineStyles?: boolean;
-  }
-
   interface Window {
     UPDATE_ANCHOR_ON_ANIMATION_FRAME?: boolean;
     ANCHOR_POSITIONING_POLYFILL_OPTIONS?: AnchorPositioningPolyfillOptions;

--- a/src/polyfill.ts
+++ b/src/polyfill.ts
@@ -443,6 +443,23 @@ async function position(rules: AnchorPositions, useAnimationFrame = false) {
   }
 }
 
+export interface AnchorPositioningPolyfillOptions {
+  // Whether to use `requestAnimationFrame()` when updating target elements’
+  // positions
+  useAnimationFrame?: boolean;
+
+  // An array of explicitly targeted elements to polyfill
+  elements?: HTMLElement[];
+
+  // Whether to exclude elements with eligible inline styles. When not defined
+  // or set to `false`, the polyfill will be applied to all elements that have
+  // eligible inline styles, regardless of whether the `elements` option is
+  // defined. When set to `true`, elements with eligible inline styles listed
+  // in the `elements` option will still be polyfilled, but no other elements
+  // in the document will be implicitly polyfilled.
+  excludeInlineStyles?: boolean;
+}
+
 function normalizePolyfillOptions(
   useAnimationFrameOrOption: boolean | AnchorPositioningPolyfillOptions = {},
 ) {

